### PR TITLE
Task eight

### PR DESCRIPTION
**Pull request by Abrams8**

### CODE REVIEW

File: src/applicationMicroservices/orderSercice/src/main/java/com/shop/ordersercice/controller/OrderController.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+import com.shop.ordersercice.entity.Order;
+import com.shop.ordersercice.service.OrderService;
+import com.shop.ordersercice.service.impl.OrderServiceImpl;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(("/orders"))
```
Comment (ALnOUA) [inline src/applicationMicroservices/orderSercice/src/main/java/com/shop/ordersercice/controller/OrderController.java]: I think these brackets are overhead, just ("/orders")

File: src/applicationMicroservices/orderSercice/src/main/java/com/shop/ordersercice/dao/MyConnection.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package com.shop.ordersercice.dao;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class MyConnection {
+    private static final String DB_URL = "jdbc:mysql://127.0.0.1:3306/shop";
+    private static final String DB_USER = "root";
+    private static final String DB_PASSWORD = "1111";
```
Comment (ALnOUA) [inline src/applicationMicroservices/orderSercice/src/main/java/com/shop/ordersercice/dao/MyConnection.java]: Move this to application.property file and inject here via @Value

File: src/main/java/com/abramchik/taskFive/controller/UserController.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
+package com.abramchik.taskFive.controller;
+
+import com.abramchik.taskFive.entity.User;
+import com.abramchik.taskFive.service.UserService;
+import com.abramchik.taskFive.service.impl.UserServiceImpl;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping(("/users"))
```
Comment (ALnOUA) [inline src/main/java/com/abramchik/taskFive/controller/UserController.java:10]: brackets

**Review by ALnOUA — CHANGES_REQUESTED** [review]

File: src/main/java/com/abramchik/taskFive/controller/UserController.java (hunk 3 above)
Comment (Abrams8) [reply src/main/java/com/abramchik/taskFive/controller/UserController.java:10]: done!

File: src/applicationMicroservices/orderSercice/src/main/java/com/shop/ordersercice/controller/OrderController.java (hunk 1 above)
Comment (Abrams8) [reply src/applicationMicroservices/orderSercice/src/main/java/com/shop/ordersercice/controller/OrderController.java]: done!

File: src/applicationMicroservices/orderSercice/src/main/java/com/shop/ordersercice/dao/MyConnection.java (hunk 2 above)
Comment (Abrams8) [reply src/applicationMicroservices/orderSercice/src/main/java/com/shop/ordersercice/dao/MyConnection.java]: I cant use @Value with static fields, but I can use setters) I rewrote this part of the code.

**Review by ALnOUA — APPROVED** [review]

**Pull request merged**